src/js/video.js

```diff
@@ -355,8 +355,8 @@ class Video {
 		const playCTA = document.createElement('div');
 		playCTA.className = `o-video__play-cta ${this.opts.placeholderHint ? 'o-video__play-cta--with-hint' : 'o-video__play-cta--without-hint'}`;
 
-		const playButtonEl = document.createElement('button');
-		playButtonEl.className = 'o-video__play-button';
+		this.playButtonEl = document.createElement('button');
+		this.playButtonEl.className = 'o-video__play-button';
 
 		this.playButtonIconEl = document.createElement('span');
```

Comment:
I don't think this needs to be added to the instance anymore as it was previously being used in the `updatePlaceholder` method, but now you're using the new `this.playButtonEl` there instead. You can probably get away with `const playButtonIconEl = ...` instead.